packages/ember-glimmer-template-compiler/lib/plugins/transform-attrs-into-args.js

```diff
@@ -38,7 +38,10 @@ TransformAttrsToProps.prototype.transform = function TransformAttrsToProps_trans
   traverse(ast, {
     PathExpression(node) {
       if (node.parts[0] === 'attrs') {
-        return b.path(node.original.substr(6));
+        let path = b.path(node.original.substr(6));
+        path.original = `@${path.original}`;
```

Comment:
Discussed with @chancancode, we are going to do an AST transform that errors on the `@` syntax.  We'll just need to ensure it runs before the one you are modifying here, but that should be fine. 

Carry on... 😎